### ISSUE TITLE
Clarification on evaluation

### ISSUE THREAD

**Issue by Sazan-Mahbub** [ml4bio/RNA-FM#10]
Hi,

Great work! 

I am trying to reproduce the SS prediction results (attached image) for [ArchiveII600](https://drive.google.com/file/d/19yVAv5ajsV5R59me4GX-Wnu1K9ps4Pnv/view?usp=drive_link) (3911 sequences) and [TS0](https://drive.google.com/file/d/1icRakk-_2afIemxyiZWKCEGaNViIVO4b/view?usp=drive_link) (1305 sequences). 

While I could exactly reproduce UFold's scores, I could not reproduce RNAFM's scores _in the same way_. I used the model weights for RNAFM from [here](https://drive.google.com/drive/folders/10YXOA5bSs8WRcZOb9Jh6fntN8NFbn8z0). 

I got the F1 score 0.666 for TS0, using "RNA-FM-ResNet_bpRNA.pth"; the paper reported 0.704. For ArchiveII600, I got 0.933 using "RNA-FM-ResNet_RNAStralign.pth"; the paper reported 0.941. 

I was wondering if the evaluation in your paper was done differently than [how UFold did it](https://github.com/uci-cbcl/UFold/blob/main/ufold_test.py) 

I'd really appreciate any help. Thank you!

<img width="835" alt="image" src="https://github.com/ml4bio/RNA-FM/assets/18265106/be1f3f70-91f6-49d4-b0c9-ee88608ae357">

**Comment by Sazan-Mahbub** [comment]
> Hi, @Sazan-Mahbub. Thank you for your interest in our work. Actually, the performances reported in this section of our preprint correspond to our fine-tuning models, whose backbone (RNA-FM) were trained together in the downstream task. However, we later updated them with our feature-based models where RNA-FMs were frozen during the training, leading to the degrading performances here. I have checked that your results are similar to but a little bit lower than ours, I think it may be caused by different threshold selection. Therefore, no worry about your metric computation, it is correct.

Hi @mydkzgj, 

Thank you for your reply and clarifications! This has been really helpful for us.

Using sigmoid and thresholding at 0.5, I am now getting F1=0.672 for TS0 and F1=0.934 for ArchiveII600 (with the same backbones mentioned before). I hope these are closer to the actual ones.

**Comment by mydkzgj** [comment]
Yeah, they're pretty much the same.
